### PR TITLE
Update Font inline docs

### PR DESCRIPTION
**Pull request by jfversluis**
### Description of Change
Adding inline docs for the Core Font objects

Related to #3960

### CODE REVIEW

**Comment by rmarinho** [comment]
Build is failing 

```
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\EmbeddedFontLoader.cs(35,11): error CS0103: The name 'Path' does not exist in the current context [D:\a\_work\1\s\src\Core\src\Core.csproj]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.3\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(807,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.3\buildTransitive\..\tools\net5.0\..\net472\XamlCompiler.exe" "obj\Release\net6.0-windows10.0.19041\\input.json" "obj\Release\net6.0-windows10.0.19041\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\src\Core.csproj]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.3\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(807,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.3\buildTransitive\..\tools\net5.0\..\net472\XamlCompiler.exe" "obj\Release\net6.0-windows10.0.20348\\input.json" "obj\Release\net6.0-windows10.0.20348\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Fonts\FontFile.cs(20,119): error CS0419: Ambiguous reference in cref attribute: 'FileNameWithExtension'. Assuming 'FontFile.FileNameWithExtension(string?)', but could have also matched other overloads including 'FontFile.FileNameWithExtension()'. [D:\a\_work\1\s\src\Core\src\Core.csproj]
```

File: src/Core/src/Fonts/FileSystemEmbeddedFontLoader.cs [hunk 1]
```diff
@@ -6,18 +6,32 @@
...
 	public class FileSystemEmbeddedFontLoader : IEmbeddedFontLoader
 	{
 		string? _rootPath;
 		readonly Func<string>? _getRootPath;
 		readonly IServiceProvider? _serviceProvider;
 
+		/// <summary>
+		/// Creates a new <see cref="FileSystemEmbeddedFontLoader"/> instance.
+		/// </summary>
+		/// <param name="rootPath">The path that will be used to copy the font to on the filesystem.</param>
```
Comment (hartez) [inline src/Core/src/Fonts/FileSystemEmbeddedFontLoader.cs:19]: Maybe "Destination filesystem path for the font"?

File: src/Core/src/Fonts/FileSystemEmbeddedFontLoader.cs [hunk 2]
```diff
@@ -6,18 +6,32 @@
...
+		/// <param name="serviceProvider">The applications <see cref="IServiceProvider"/>.
```
Comment (hartez) [inline src/Core/src/Fonts/FileSystemEmbeddedFontLoader.cs:20]: "applications" -> "application's"

File: src/Core/src/Fonts/FileSystemEmbeddedFontLoader.cs [hunk 3]
```diff
@@ -6,18 +6,32 @@
...
 		public FileSystemEmbeddedFontLoader(string rootPath, IServiceProvider? serviceProvider = null)
 		{
 			_rootPath = rootPath ?? throw new ArgumentNullException(nameof(rootPath));
 			_serviceProvider = serviceProvider;
 		}
 
+		/// <summary>
+		/// Creates a new <see cref="FileSystemEmbeddedFontLoader"/> instance.
+		/// </summary>
+		/// <param name="getRootPath">Function that is executed to retrieve the path that will be used to copy the font to on the filesystem.</param>
```
Comment (hartez) [inline src/Core/src/Fonts/FileSystemEmbeddedFontLoader.cs:32]: Maybe "Function which retrieves the destination filesystem path for the font."?

File: src/Core/src/Fonts/FontManager.Windows.cs [hunk 4]
```diff
@@ -25,18 +25,27 @@ public class FontManager : IFontManager
 		readonly IFontRegistrar _fontRegistrar;
 		readonly IServiceProvider? _serviceProvider;
 
+		/// <summary>
+		/// Creates a new <see cref="EmbeddedFontLoader"/> instance.
+		/// </summary>
+		/// <param name="fontRegistrar">A <see cref="IFontRegistrar"/> instance to retrieve details from about registered fonts.</param>
```
Comment (hartez) [inline src/Core/src/Fonts/FontManager.Windows.cs:31]: Maybe "An IFontRegistrar instance for retrieving details about the registered fonts."?

**Review by hartez — CHANGES_REQUESTED** [review]
Some wording and punctuation stuff.

**Review by hartez — APPROVED** [review]

**Review by rmarinho — APPROVED** [review]

**Pull request merged**